### PR TITLE
explorer: home page update

### PR DESCRIPTION
**Pull request by pchrysochoidis**
## Description 

* adds Accounts graph card
* adds transactions graph card
    * this doesn't work on testnet because of API issues (confirmed with @mystie711 that this is fine for testnet)
    * Screenshot 2023-07-11 at 12 46 53

* updates home page UI
* changes the header to new UI

https://github.com/MystenLabs/sui/assets/10210143/1e4810e1-ee7b-47f9-a2cd-efd0a4b4891d

closes APPS-1256
closes APPS-1356
closes APPS-1368

## Test Plan 

👀 

---
If your changes are not user-facing and not a breaking change, you can skip the following section. Otherwise, please indicate what changed, and then add to the Release Notes section as highlighted during the release process.

### Type of Change (Check all that apply)

- [ ] protocol change
- [ ] user-visible impact
- [ ] breaking change for a client SDKs
- [ ] breaking change for FNs (FN binary must upgrade)
- [ ] breaking change for validators or node operators (must upgrade binaries)
- [ ] breaking change for on-chain data layout
- [ ] necessitate either a data wipe or data migration

### CODE REVIEW

**Comment by pchrysochoidis** [comment]
merged https://github.com/MystenLabs/sui/pull/12836 & https://github.com/MystenLabs/sui/pull/12834 in this one for easier testing

**Review by mystie711 — APPROVED** [review]
woot! lgtm.

**Comment by pchrysochoidis** [comment]
Updated loading and no data indicator spacing and border radius
Screenshot 2023-07-11 at 16 12 10
Screenshot 2023-07-11 at 16 15 56

**Comment by pchrysochoidis** [comment]
Also removed the refresh token price button as per @mystie711 request
Screenshot 2023-07-11 at 18 26 14

**Comment by pchrysochoidis** [comment]
Sui icon in token price card now is center aligned

| before | after |
| -- | -- |
| Screenshot 2023-07-11 at 19 14 43  | Screenshot 2023-07-11 at 19 16 29 | 

cc @mystie711

**Comment by pchrysochoidis** [comment]
renamed transactions graph title to Transaction Blocks

Screenshot 2023-07-11 at 20 10 45

File: apps/explorer/src/components/Layout/index.tsx [hunk 1]
```diff
@@ -58,21 +61,31 @@ export function Layout() {
 			>
 				<RpcClientContext.Provider value={jsonRpcProvider}>
 					<NetworkContext.Provider value={[network, setNetwork]}>
-						<div className="w-full">
+						<div className={clsx('w-full', isGradientVisible && 'bg-[#D2EBFA]')}>
```
Comment (plam-ml) [inline apps/explorer/src/components/Layout/index.tsx:64]: small nit here, but can we move this into the config file? `#D2EBFA`

**Review by plam-ml — APPROVED** [review]
lgtm, tho 1 question regarding the bg gradient. That will show up eventually on other pages. Should we make it easier to know which page will receive the gradient as we migrate to the updated UI, instead of calling `setIsGradientVisible` as it looks like here?

**Comment by pchrysochoidis** [comment]
> lgtm, tho 1 question regarding the bg gradient. That will show up eventually on other pages. Should we make it easier to know which page will receive the gradient as we migrate to the updated UI, instead of calling `setIsGradientVisible` as it looks like here?

So this used a Portal that was actually populating the gradient content and was calling the setGradientVisible function. So each page that had gradient content was using a `GradientContentConponent` that was handling the portal stuff and this was happening automatically. But since this forces unnecessary renders and maybe a bit confusing I updated it to add a PageLayout component that every page has to use to provide main and optionally gradient content.

cc @plam-ml

**Pull request merged**